File: problems/python/bfs/construct_shortest_path.py
"""
Given 3 inputs: a start city, an end city, and a list of roads, return the shortest path (as an array) from the start city to the end city.
"""

# TODO: write your code here

start_city = 5
end_city = 10
roads = [[5,7], [5,3], [7,6], [7,4], [3,9], [6,4], [4,10], [4,9]]

print(f"Your answer: {get_shortest_path(start_city, end_city, roads)}")
print(f"Correct answer: {2}")
print()

File: problems/javascript/bfs/constructShortestPath.js
/*
Given 3 inputs: a start city, an end city, and a list of roads, return the shortest path (as an array) from the start city to the end city.
*/

const { Queue } = require('../../utils/queue');

// TODO: write your code here

const startCity = 5;
const endCity = 10;
const roads = [
    [5, 7],
    [5, 3],
    [7, 6],
    [7, 4],
    [3, 9],
    [6, 4],
    [4, 10],
    [4, 9],
];

console.log(`Your answer: ${getShortestPath(startCity, endCity, roads)}`);
console.log(`Correct answer: ${2}`);
console.log();
